src/main/scala/io/github/pauljamescleary/petstore/infrastructure/repository/doobie/DoobieUserRepositoryInterpreter.scala

```diff
@@ -53,7 +53,7 @@ private object UserSQL {
   """.query
 }
 
-class DoobieUserRepositoryInterpreter[F[_]](val xa: Transactor[F])(implicit M: MonadError[F, Throwable])
+class DoobieUserRepositoryInterpreter[F[_]](val xa: Transactor[F])(implicit M: Monad[F])
```

Comment:
This is fine. Going back to our discussion about parameters vs context bounds, I think this is a case when context bounds are appropriate.

Reply:
Note neatly using MonadError would have required kind-projector.